### PR TITLE
WEBUI-96: allow promotion.yaml to accept ref branch and dry run as input 

### PR DESCRIPTION
**Pull request by Gabez0r**
Related https://github.com/nuxeo/nuxeo-elements/pull/322.

As a bonus, also creating a github release.

### CODE REVIEW

**Review by mnixo — DISMISSED** [review]

**Comment by Gabez0r** [comment]
This is how the workflow form looks like:
image

File: .github/workflows/promote.yaml [hunk 1]
```diff
@@ -92,17 +117,30 @@ jobs:
           npm version $NEW_SNAPSHOT --no-git-tag-version
           popd
 
+      - if: ${{ github.event.inputs.referenceBranch == 'master' }}
+        run: |
+          NEW_ELEMENTS_VERSION="^$NEW_VERSION-rc"
+          npm i @nuxeo/nuxeo-elements@$NEW_ELEMENTS_VERSION --save-prefix="^"
```
Comment (nelsonsilva) [inline .github/workflows/promote.yaml:123]: the caret is the default prefix, right ?
also, if we're going with the caret i guess in practice we don't really have to do anything in the master branch, right ?

Comment (Gabez0r) [reply .github/workflows/promote.yaml:123]: WDYM by _we don't really have to do anything_? We need to install it anyway, in order to update the version. But perhaps we don't need the `--save-prefix="^"` part?

Comment (nelsonsilva) [reply .github/workflows/promote.yaml:123]: nevermind, forgot about the pre-release version we want to install

**Review by nelsonsilva — DISMISSED** [review]

**Review by mnixo — APPROVED** [review]

**Review by richardsd — APPROVED** [review]
I'll trust you on this one 😬 
This is going to be available here in Github right?

**Comment by Gabez0r** [comment]
Yes, on the Actions tab.

**Review by nelsonsilva — APPROVED** [review]

**Pull request merged**